README.md

```diff
@@ -71,6 +71,7 @@ by the client itself.
 
 ### Events
 
+- client-initialized: Triggers when the drop-zone client have been initialized. You can use this event to do further customizations on the client in the higher ordered component.
```

Comment:
`Is triggered when the drop-zone client has been initialised. Use this event to customise the client.`
What do you mean by `higher ordered component` in the comment?

Reply:
the component which is using the drop-zone component in its template

Reply:
But the parent component does not have to be a HOC? This term in the readme is confusing for other developers IMO. The plugin does not depend on HOC, so I would no mention it here.

Reply:
Why not? A parent component is always higher ordered in the hierarchy of components.

But I'll update the text. I agree that it's probably a little confusing here 👍